### PR TITLE
Translations update from Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Weblate](https://hosted.weblate.org/projects/sdg-translations/general/) for SDG Translations/General.

Current translation status:

Weblate translation status

### CODE REVIEW

File: translations/fr/general.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 goal: Objectif
-goals: Buts
+goals: Objectifs
```
Comment (LucyGwilliamAdmin) [inline translations/fr/general.yml:2]: @brockfanning this looks right to me. Buts seems to mean goals in the scoring sense e.g. points.

Disclaimer: I don't speak French

File: translations/fr/calendar.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 January: janvier
 February: février
-March: Mars
+March: mars
```
Comment (LucyGwilliamAdmin) [inline translations/fr/calendar.yml:3]: Looking [here](https://www.fluentu.com/blog/french/french-months-of-the-year/) all months should begin with lower case so think this looks good

**Review by LucyGwilliamAdmin — COMMENTED** [review]
@brockfanning I've left some comments as to why I think this can be merged. I also think Mai should be mai for May

**Comment by brockfanning** [comment]
Sounds good!

**Comment by brockfanning** [comment]
Sounds good!

**Pull request merged**